### PR TITLE
[Minor] [ML] override toString of AttributeGroup

### PR DESCRIPTION
**Pull request by ericl**
This makes equality test failures much more readable.

@mengxr

### CODE REVIEW

**Comment by mengxr** [comment]
LGTM pending Jenins

File: mllib/src/main/scala/org/apache/spark/ml/attribute/AttributeGroup.scala [hunk 1]
```diff
@@ -183,6 +183,8 @@ class AttributeGroup private (
     sum = 37 * sum + attributes.map(_.toSeq).hashCode
     sum
   }
+
+  override def toString = toMetadata.toString
```
Comment (mengxr) [inline mllib/src/main/scala/org/apache/spark/ml/attribute/AttributeGroup.scala]: okay, forgot explicit type for public methods

**Comment by ericl** [comment]
Oops, fixed.

**Comment by mengxr** [comment]
Merged into master. Thanks!

**Pull request closed without merging**